editor.planx.uk/src/components/EditorNavMenu/EditorNavMenu.tsx

```diff
@@ -99,31 +100,37 @@ function EditorNavMenu() {
       title: "Settings",
       Icon: TuneIcon,
       route: `/${teamSlug}/general-settings`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
     },
     {
       title: "Design",
       Icon: PaletteIcon,
       route: `/${teamSlug}/design`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
     },
     {
       title: "Team members",
       Icon: GroupIcon,
       route: `/${teamSlug}/members`,
-      accessibleBy: ["platformAdmin", "teamEditor"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor"],
+    },
+    {
+      title: "Subscription",
+      Icon: CurrencyPoundIcon,
+      route: `/${teamSlug}/subscription`,
+      accessibleBy: ["platformAdmin", "teamAdmin"],
     },
     {
       title: "Feedback",
       Icon: RateReviewIcon,
       route: `/${teamSlug}/feedback`,
-      accessibleBy: ["platformAdmin", "teamEditor", "demoUser"],
+      accessibleBy: ["platformAdmin", "teamAdmin", "teamEditor", "demoUser"],
```

Comment:
We might need to change the way that `getUserRoleForCurrentTeam()` works to return an array now that we have multiple team-based roles - most (all?) `teamAdmins` will already have `teamEditor` permissions.

Reply:
I've actually just also spotted a bug in the current implementation which also means we should return an array.

🐛 If a user is both an `analyst` and a `teamAdmin`, they cannot view non-analyst links (such as `/subscription`.